docs/transpile/rest-api-transpilation.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Transpilation using REST APIs
+description: How to transpile quantum circuits using REST APIs.
+---
+
+# Transpilation using REST APIs
+
+The circuit provided will need to be mapped to instructions fitting to the hardware basis instructions of the QPU, which is a process known as Transpilation.
+
+* You can either transpile [locally via Qiskit](https://docs.quantum.ibm.com/transpile/defaults-and-configuration-options) before generating the QASM string, or:
+
+* Use the [Cloud Transpiler API](https://cloud-transpiler.quantum.ibm.com/docs)
+
+<Admonition type="note">
+    [Transpilation is necessary](https://docs.quantum.ibm.com/announcements/product-updates/2024-02-14-qiskit-runtime-primitives-update) prior to submitting a circuit to IBM Quantum backends. 
+</Admonition>
+
+
+### Query the Cloud Transpilation service
```

Comment:
Following accessibility guidelines I think this should be an H2 not an H3. In general it's recommend to follow the hierarchy h1->h2->h3->h4, instead of jumping from h1 directly to h3